src/utils/Constants.js

```diff
@@ -58,6 +58,14 @@ export const PRIORITIES = [
     label: "no priority",
   },
 ];
+
+export const NEWPRIORITIES = {
+  'low': {'label':'low', 'color_code':'#00A031'},
```

Comment:
Make 'label'  to label.
'color_code' to color_code.

Essentially remove them as string form